src/main/resources/swagger/agora.yaml

```diff
@@ -1322,7 +1322,133 @@ paths:
           schema:
             type: object
             $ref: '#/definitions/Error'
-  /ga4gh/tools/{id}/versions/{version-id}/{type}/descriptor:
+
```

Comment:
please read PR description - I know some of this swagger is incorrect and needs updating.